src/components/Link.test.tsx

```diff
@@ -0,0 +1,12 @@
+import Link from "./Link";
+import React from "react";
+import { create } from "react-test-renderer";
+
+it("renders correctly with all props", () => {
+  const component = create(
+    <Link href="/" className="class1">
```

Comment:
```suggestion
    <Link className="class1" href="/">
```

It's a bit pedantic, but I think it does help readability.